admin_manual/enterprise/index.rst

```diff
@@ -7,6 +7,7 @@ Enterprise Features
 
    installation/index
    firewall/index
+   security/index
```

Comment:
Hmm, I think we should directly call it "ransomware protection" or integrate "firewall" in the new security section as it would be inconsistent otherwise.

Reply:
Ok, that makes sense.

Reply:
I've revised the location, structure, and naming of the file in the most recent commit. Does that fit better?